js/scripts.js

```diff
@@ -0,0 +1,7 @@
+//Task: area of triangle
+
+var a = prompt('Podaj wartość a'),
+	h = prompt('Podaj wartość h'),
+	triangleArea = a*h/2;
```

Comment:
Ok dzięki. A możesz wyjaśnić dlaczego? Na kodilla sprzedawali to tak, że jest krócej.